eng/Subsets.props

```diff
@@ -121,9 +121,10 @@
 
   <PropertyGroup>
     <!-- CLR NativeAot only builds in a subset of the matrix -->
+    <_IsCommunityCrossArchitecture Condition="'$(CrossBuild)' == 'true' and ('$(TargetArchitecture)' == 'loongarch64' or '$(TargetArchitecture)' == 'riscv64')">true</_IsCommunityCrossArchitecture>
```

Comment:
I am not sure we want to have NAOT disabled here by default in the long run, but it is fine for now.

Reply:
https://github.com/dotnet/runtime/pull/105004 should re-enable it along with the packs subset.